qt/TorrentFilter.cc

```diff
@@ -161,37 +159,36 @@ bool TorrentFilter::lessThan(QModelIndex const& left, QModelIndex const& right)
 
         if (val == 0)
         {
-            val = tr_compare_3way(a->percentComplete(), b->percentComplete());
+            val = a->compareSeedProgress(*b);
         }
 
         if (val == 0)
         {
-            val = a->compareSeedProgress(*b);
+            val = tr_compare_3way(a->getActivity(), b->getActivity());
         }
```

Comment:
Actually, that changes seems to cause a regression in sorting by progress.

So self-assigning to me to figure it out.

Reply:
c17caf9f08df7a7bc524807a3752d9b9b943eb29